### PR TITLE
USBDevice.controlTransferIn/Out et al - confusing text

### LINKED ISSUE #22652: Conflicting information in `USBDevice.controlTransferIn()` and `USBDevice.controlTransferOut()`
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/USBDevice

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/API/USBDevice#:~:text=USBDevice.controlTransferIn()

### What information was incorrect, unhelpful, or incomplete?

On the main page, it says

> [USBDevice.controlTransferIn()](https://developer.mozilla.org/en-US/docs/Web/API/USBDevice/controlTransferIn) Experimental
Returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves with a [USBInTransferResult](https://developer.mozilla.org/en-US/docs/Web/API/USBInTransferResult) when a command or status operation has been transmitted **to the USB device.**

On the [USBDevice.controlTransferIn()](https://developer.mozilla.org/en-US/docs/Web/API/USBDevice/controlTransferIn) page it says:

> The controlTransferIn() method of the [USBDevice](https://developer.mozilla.org/en-US/docs/Web/API/USBDevice) interface returns a [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves with a [USBInTransferResult](https://developer.mozilla.org/en-US/docs/Web/API/USBInTransferResult) when the result of a command or status request has been received **from the USB device**.

I'm having difficulty in understanding which is correct.

The same is applicable in the `USBDevice.controlTransferOut()` section as well.

Also please verify all the `USBDevice.xxxxxOut` and `USBDevice.xxxxxIn` methods in that page.

### What did you expect to see?

Information in both pages should be in sync and correct

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/usbdevice`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/USBDevice
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/usbdevice/index.md
* Last commit: https://github.com/mdn/content/commit/758b1253ca3478c06a1a575965fd32ac1e2bdda2
* Document last modified: 2022-11-15T16:45:25.000Z

</details>

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #22652

The text was incorrect and confusing, and did not match what "In" means. This applied to the in/out and iso transfer methods.
Also some minor layout fixes.

Note, I did not check everything - just the stuff to do with transfers.

### CODE REVIEW

File: files/en-us/web/api/usbdevice/controltransferin/index.md [hunk 1]
```diff
@@ -17,10 +17,7 @@ browser-compat: api.USBDevice.controlTransferIn
 
 {{APIRef("WebUSB API")}}{{SeeCompatTable}}
 
-The **`controlTransferIn()`** method of the
-{{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a
-{{domxref("USBInTransferResult")}} when the result of a command or status request has
-been received from the USB device.
+The **`controlTransferIn()`** method of the {{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a {{domxref("USBInTransferResult")}} when a command or status request has been transmitted to (received by) the USB device.
```
Comment (Josh-Cena) [inline files/en-us/web/api/usbdevice/controltransferin/index.md:20]:
```suggestion
The **`controlTransferIn()`** method of the {{domxref("USBDevice")}} interface returns a {{jsxref("Promise")}} that resolves with a {{domxref("USBInTransferResult")}} when a command or status request has been transmitted to (received by) the USB device.
```

And probably update other occurrences as well

File: files/en-us/web/api/usbdevice/controltransferout/index.md [hunk 2]
```diff
@@ -17,10 +17,7 @@ browser-compat: api.USBDevice.controlTransferOut
 
 {{APIRef("WebUSB API")}}{{SeeCompatTable}}
 
-The **`controlTransferOut()`** method of the
-{{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a
-{{domxref("USBOutTransferResult")}} when a command or status operation has been
-transmitted to the USB device.
+The **`controlTransferOut()`** method of the {{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a {{domxref("USBOutTransferResult")}} when a command or status operation has been transmitted from the USB device.
```
Comment (hamishwillee) [inline files/en-us/web/api/usbdevice/controltransferout/index.md:20]:
```suggestion
The **`controlTransferOut()`** method of the {{domxref("USBDevice")}} interface returns a {{jsxref("Promise")}} that resolves with a {{domxref("USBOutTransferResult")}} when a command or status operation has been transmitted from the USB device.
```

File: files/en-us/web/api/usbdevice/isochronoustransferin/index.md [hunk 3]
```diff
@@ -17,10 +17,7 @@ browser-compat: api.USBDevice.isochronousTransferIn
 
 {{APIRef("WebUSB API")}}{{SeeCompatTable}}
 
-The **`isochronousTransferIn()`** method of the
-{{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a
-{{domxref("USBIsochronousInTransferResult")}} when time sensitive information has been
-transmitted received from the USB device.
+The **`isochronousTransferIn()`** method of the {{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a {{domxref("USBIsochronousInTransferResult")}} when time sensitive information has been transmitted to (received by) the USB device.
```
Comment (hamishwillee) [inline files/en-us/web/api/usbdevice/isochronoustransferin/index.md:20]:
```suggestion
The **`isochronousTransferIn()`** method of the {{domxref("USBDevice")}} interface returns a {{jsxref("Promise")}} that resolves with a {{domxref("USBIsochronousInTransferResult")}} when time sensitive information has been transmitted to (received by) the USB device.
```

File: files/en-us/web/api/usbdevice/isochronoustransferout/index.md [hunk 4]
```diff
@@ -18,9 +18,7 @@ browser-compat: api.USBDevice.isochronousTransferOut
 {{APIRef("WebUSB API")}}{{SeeCompatTable}}
 
 The **`isochronousTransferOut()`** method of the
-{{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a
-{{domxref("USBIsochronousOutTransferResult")}} when time sensitive information has been
-transmitted to the USB device.
+{{domxref("USBDevice")}} interface returns a {{jsxref("promise")}} that resolves with a {{domxref("USBIsochronousOutTransferResult")}} when time sensitive information has been transmitted from the USB device.
```
Comment (hamishwillee) [inline files/en-us/web/api/usbdevice/isochronoustransferout/index.md:21]:
```suggestion
{{domxref("USBDevice")}} interface returns a {{jsxref("Promise")}} that resolves with a {{domxref("USBIsochronousOutTransferResult")}} when time sensitive information has been transmitted from the USB device.
```

File: files/en-us/web/api/usbdevice/isochronoustransferin/index.md [hunk 5]
```diff
@@ -37,8 +34,7 @@ isochronousTransferIn(endpointNumber, packetLengths)
 
 ### Return value
 
-A {{jsxref("promise")}} that resolves with a
-{{domxref("USBIsochronousInTransferResult")}}
+A {{jsxref("promise")}} that resolves with a {{domxref("USBIsochronousInTransferResult")}}.
```
Comment (hamishwillee) [inline files/en-us/web/api/usbdevice/isochronoustransferin/index.md:37]:
```suggestion
A {{jsxref("Promise")}} that resolves with a {{domxref("USBIsochronousInTransferResult")}}.
```

**Comment by hamishwillee** [comment]
Fixed the promise casing.

**Review by Elchi3 — APPROVED** [review]
👍

**Pull request merged**